website/docs/compute/deployment/other-deployments/minikube.md

```diff
@@ -0,0 +1,231 @@
+
+Under the Builder Container:
+
+```yaml
+- name: builder
+          image: suborbital/scc-builder:v0.3.1
+          command: ["builder"]
+
+          ports:
+            - containerPort: 8080
+            - containerPort: 8443
+
+          env:
+            - name: SCC_DOMAIN
+              value: "<YOUR_NGROK_DOMAIN>"
+
+            - name: SCC_TLS_PORT
+              value: "8443"
+
+            - name: SCC_LOG_LEVEL
+              value: "info"
+
+            - name: SCC_CONTROL_PLANE
+              value: "scc-controlplane-service:8081"
+
+          volumeMounts:
+            - mountPath: "/home/scn"
+              name: controlplane-storage
+```
+
+Delete the following line:
+
+`delete containerPort: 8443`
+
+Delete the following key-value pair:
+
+```yaml
+- name: SCC_DOMAIN
+  value: "<YOUR_NGROK_DOMAIN>"
+```
+
+Replacing the following key-value pair:
+
+```yaml
+- name: SCC_TLS_PORT
+  value: "8443"
+```
+
+With the following:
+
+```yaml
+name: SCC_HTTP_PORT
+value: "8080"
+```
+
+Under the `scc-builder-service`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: suborbital
+  name: scc-builder-service
+spec:
+  selector:
+    app: scc-controlplane
+  ports:
+    - protocol: TCP
+      name: challenge
+      port: 80
+      targetPort: 8080
+    - protocol: TCP
+      name: https
+      port: 443
+      targetPort: 8443
+  type: LoadBalancer
+```
+
+Our builder service no longer needs to expose HTTPS ports as ngrok will forward both HTTP and HTTPS traffic to port 80.
+
+Remove the following lines:
+
+```yaml
+- protocol: TCP
+  name: https
+  port: 443
+  targetPort: 8443
+```
+
+### 6. Deploy to your cluster
+
+Run the following Subo command to deploy Compute to your cluster:
+
+```bash
+subo compute deploy core
+```
+
+### 7. Setup minikube tunneling
+
+Let’s tell minikube to forward requests to port 80 to our cluster!
+
+```bash
+minikube tunnel
+```
+
+### 8. Create an editor token
+
+In order to test our editor, we’re going to come up with a function name, and create a token so we can access it!
+
+This can only be done as an [API call](https://docs.suborbital.dev/compute/integrate-the-function-editor/code-editor) from within your cluster. Since we’re currently not running an app in our cluster, we’ll just make the call from within!
+
+First, we’ll need the name of our control plane pod:
+
+```bash
+kubectl get pod -n suborbital
+```
+
+Your output will look something like this:
+
+```bash
+NAME                                           READY   STATUS    RESTARTS   AGE
+scc-atmo-deployment-7bfb9d76c6-sv5dr           1/1     Running   0          27s
+scc-controlplane-deployment-5699f779f7-xmkhr   2/2     Running   0          27s
+```
+
+Let’s take that full name of our `scc-controlplane-deployment` pod and start a bash session inside it:
+
+```bash
+kubectl exec -n suborbital -it scc-controlplane-deployment-<REST OF POD CODENAME> -- bash
+```
+
+Would you look at that, we’re inside our cluster now!
```

Comment:
This feels a bit off of our usual tone in the docs.

Reply:
This isn't "mine", as such; it was merged in PR https://github.com/suborbital/docs/pull/158 (I think it's showing as a change here because I had a stub for it before the other was merged and had to swap out the stub for the merged one. Probably there's a better way to do that, my bad).

I do like an aggressively upbeat and fun tone, though. 😊 If you'd rather we stay more neutral than cheerleader, I can change it ofc!